daemon/logger/jsonfilelog/read.go

```diff
@@ -129,7 +134,23 @@ func followLogs(f *os.File, logWatcher *logger.LogWatcher, notifyRotate chan int
 	if err != nil {
 		logWatcher.Err <- err
 	}
-	defer fileWatcher.Close()
+	defer func() {
+		f.Close()
```

Comment:
this seems to already be closed by https://github.com/docker/docker/pull/21301/files#diff-f06ce8e825525a9e55c84c084759dd01R66